src/NewRelic.Telemetry/TelemetryConfiguration.cs

```diff
@@ -77,11 +77,6 @@ class TelemetryConfiguration
         /// </summary>
         public string? ServiceName { get; set; }
 
-        /// <summary>
-        /// Identifies the source of information that is being sent to New Relic.
-        /// </summary>
-        public string? InstrumentationProvider { get; set; }
```

Comment:
Since this isn't a spec'd configuration option, it's nice that we're simplifying things.